### DISCUSSION TITLE
Does ASReview use information from full text for article ranking/prioritizing?

### DISCUSSION THREAD

**Discussion by YNoteboom** [asreview/asreview#1398] [Q&A] [answered]
Hi, 

Hopefully you can help us with this, could not find the awnser yet. 
We are considering using ASReview to support our review proces. During our preliminaire search/ building our search strategy we found out that some articles don't mention the information we search for in the title/abstract but it is 'hidden' in the full text. So I was wondering whether ASReview also uses information from the full text to rank the articles on 'relevant' to least-relevant? Our does it only uses text from the title and abstract? 
If anything is unclear I'm happy to clarify. 
Thank you!

**Comment by YNoteboom** [comment, marked as answer]
I Think this is answered in #1384 so discussion can be closed or are there any other suggestions on this?

**Comment by FelixWdm** [comment]
Hi @YNoteboom, when you find articles where the title or abstract don't exactly match, it's most likely the index-keywords. In PubMed, MeSH terms, they are linked to the article and are added based on the full text. If you share the search and platform you searched in, I can check it in more detail.
